dev-db/influxdb/influxdb-2.7.1.ebuild

```diff
@@ -180,6 +180,10 @@ COMMON_DEPEND="
 
 DEPEND="${COMMON_DEPEND}"
 
+RDEPEND="
+	cli? ( dev-db/influx-cli )
```

Comment:
If we're going to have this as a transition thing, we should default on +cli and also add a warning in pkg_postinst like we did for Docker.

I don't think we should necc. keep this long-term, as it's a runtime-only USE flag (https://projects.gentoo.org/qa/policy-guide/use-flags.html#pg0802).

Reply:
Yeah, I think that should be OK. It's good enough to help most people.